PINCache/PINDiskCache.h

```diff
@@ -1,5 +1,6 @@
 //  PINCache is a modified version of TMCache
 //  Modifications by Garrett Moon
+//  Modifications by Sebastien Villar, Microsoft
```

Comment:
Sure I can definitely remove it. I thought I needed that for compliance but it's fine :S